zoo/services/models.py

```diff
@@ -26,6 +26,13 @@ class Impact(Enum):
     EMPLOYEES = "employees"
 
 
+class Tier(Enum):
```

Comment:
I'm not a huge fan of having this fully hardcoded, as I can imagine other users for which this tiers might mean nothing 😕 

And also we don't have any description for what every tier means.

I think it might make more sense to add a new `Tier` model, with level and description, and we can enter the tiers through the admin panel for now.

Reply:
and the `tier` in the service model will just be a foreign key to the `tier` table.

Reply:
and we should probably keep it nullable, as not every Zoo user might need to use the tiers

Reply:
I agree, that’s pretty much what I reverted.

Reply:
oh... 😓